pkgs/top-level/all-packages.nix

```diff
@@ -1251,6 +1251,8 @@ in
 
   pass = callPackage ../tools/security/pass { };
 
+  pass-nodmenu = callPackage ../tools/security/pass { dmenuSupport = false; };
```

Comment:
you most likely want to do `pass = pass-nodmenu` as it's done for `pass-wayland`: otherwise, you'll use the wrong pass again when doing `pass-nodmenu.withExtensions`.